testsuite/features/step_definitions/api_common.rb

```diff
@@ -500,13 +500,13 @@ def wait_action_complete(actionid, timeout: DEFAULT_TIMEOUT)
   step %(I should get status "#{status}" for system "#{get_system_id(node)}")
 end
 
-Then(/^I should get the test channel$/) do
+Then(/^I should get the test base channel$/) do
   arch = `uname -m`
   arch.chomp!
   channel = if arch != 'x86_64'
               'fake-base-channel-i586'
```

Comment:
weird code... it is executed on the controller... and we would have i586 controllers ???

Reply:
Improved, thanks for that remark.